src/components/00-materials/README.md

```diff
@@ -88,13 +88,30 @@ npm install @svgr/webpack --save-dev
 
 ## Contribution
 
-### Process of adding a new icon/image
+### Process of adding a new image
 
 1. Involve a designer by sending him the new SVG in question, unless the designer already has the file. He will review it and possibly change some things, for example by adjusting inner padding etc. in order to guarantee overall quality.
-2. Remove unnecessary code inside the SVG:
+1. Remove unnecessary code inside the SVG:
    - a. `data-name="Layer 2"` # layer info for vector drawing software
    - b. `<path fill="#fff" d="M0 0h96v96H0z"/>` # outer bounding box filled with white background
-3. - a. use the internal [svgo](https://github.com/svg/svgo) -invoking scripts `npm run build-icons` or `npm run build-images` from materials itself to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `icons-raw` or `images-raw`
+1. - a. use the internal [svgo](https://github.com/svg/svgo) -invoking script `npm run build-images` from the materials-folder itself to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `images-raw`
    - b. alternatively use the [online SVG optimizer](https://jakearchibald.github.io/svgomg/) for the same purpose
-4. Manually review the SVGs produced in the last step: add or edit &lt;path&gt; attributes, setting `fill="currentColor"` and/or `strokes="currentColor"` as appropriate. Remove unnecessary attributes.
-5. Run `npm run build` to generate a js file for each svg file found. The generated files can be found in the `icons`and `images` folder.
+1. Manually review the SVGs produced in the last step: add or edit &lt;path&gt; attributes, setting `fill="currentColor"` and/or `strokes="currentColor"` as appropriate. Remove unnecessary attributes.
+1. Run `npm run build` to generate a js file for each svg file found. The generated files can be found in the `images` folder.
+
+### Updating materials (UNIX only)
+
+All our icons in the "icons" section are the material icons from google (and nothing else). To update the material icons from google (https://github.com/google/material-design-icons/releases), follow these steps.
+
+1. Remove the contents of the `icons-raw` folder.
+1. Download the newest version of material icons as a ZIP file to your download folder.
+1. Extract the material icons zip file into the downloads folder.
+1. Rename the extracted folder to `material-design-icons`.
+1. Open a terminal and `cd` into the materials root folder (`src/components/00-materials/`).
+1. Run `sh material-importer.sh`.
+1. Run a search on only the folder `src/components/00-materials/icons-raw` and search for `<path` (whitespace at the end).
+1. Replace all occurences with `<path fill="currentColor"` (whitespace at the end).
+1. Run a search on only the folder `src/components/00-materials/icons-raw` and search for `<circle` (whitespace at the end).
+1. Replace all occurences with `<circle fill="currentColor"` (whitespace at the end).
```

Comment:
Is it possible to replace that with the script?

Reply:
it would be possible, but that takes a looooot of effort. This could be done in the future, but at the moment, this is a pretty straight forward way.